### PR TITLE
list listening ports in status report

### PR DESCRIPTION
**Pull request by rmg**
Connect to strongloop-internal/scrum-nodeops#507

### CODE REVIEW

File: bin/sl-meshctl.js [hunk 1]
```diff
@@ -223,6 +224,7 @@ function printServiceStatus(service) {
         proc.displayId,
         proc.pid,
         proc.workerId,
+        _.pluck(proc.listeningSockets, 'port').join(','),
```
Comment (sam-github) [inline bin/sl-meshctl.js]: Will this properly ignore unix domain and UDP sockets? unix domain won't have a port, not sure if pluck will return undefined or ignore that one, but UDP will have a port, I  think.

**Comment by sam-github** [comment]
basically LGTM, though I'm not sure its robust against non-TCP ports. because the node docs are so terrible, a manual test would probably be in order

**Comment by rmg** [comment]
I wasn't going to worry about the domain socket case because it seems even more unlikely than listening on multiple ports for our theoretical user.. but I've fixed it now. PTAL.

**Comment by sam-github** [comment]
LGTM

**Pull request merged**